src/main.cpp

```diff
@@ -133,6 +135,8 @@ int main(int argc, char** argv) {
          "Table tailer barrier type. EPOCH=0, GCI=1")
         ("reindex", po::value<bool>(&reindex)->default_value(reindex),
          "initialize an empty index with all metadata")
+        ("reindex_of", po::value<std::string>(&reindex_of)->default_value(reindex_of),
+         "define index to initialize")
```

Comment:
write in the description the possible values [project, featurestore]

Reply:
fixed